### PR TITLE
Add cellranger to multiqc report

### PR DESCRIPTION
**Pull request by grst**
Since MultiQC v1.15, there's a cellranger module. 
This PR activates this functionality for the cellranger branch of the pipeline. 

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/scrnaseq/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/scrnaseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

Partly addresses #80

### CODE REVIEW

File: workflows/scrnaseq.nf [hunk 1]
```diff
@@ -193,6 +194,9 @@ workflow SCRNASEQ {
         ch_versions = ch_versions.mix(CELLRANGER_ALIGN.out.ch_versions)
         ch_mtx_matrices = ch_mtx_matrices.mix(CELLRANGER_ALIGN.out.cellranger_out)
         ch_star_index = CELLRANGER_ALIGN.out.star_index
+        ch_multiqc_cellranger = CELLRANGER_ALIGN.out.cellranger_out.map{
+            meta, outs -> outs.findAll{ it -> it.name == "web_summary.html"}
```
Comment (SPPearce) [inline workflows/scrnaseq.nf:198]: Why is this part necessary? Is this something that could be changed in the `CELLRANGER_ALIGN` module to make it play nicely with multiqc out of the box?

**Review by SPPearce — APPROVED** [review]
I have a minor question about what the map is doing, but otherwise all fine.

File: workflows/scrnaseq.nf (hunk 1 above)
Comment (grst) [reply workflows/scrnaseq.nf:198]: It's mostly to make MultiQC more efficient, because it only gets the files it needs (the full cellranger output dir contains a lot of files)
